### PR TITLE
Handle crashes

### PR DESCRIPTION
**Pull request by reaganlo**
Fixes Issue: #417

Changes proposed in this pull request:
- Introduces Panic() to handle crashes
- Removed "model" and "installReboot" variables from Gui struct as they are not being used.
- Updated log level from warning to errors wherever applicable.

### CODE REVIEW

File: gui/window.go [hunk 1]
```diff
@@ -781,6 +782,16 @@ func (window *Window) SetScanMedia(scannedMedia []*storage.BlockDevice) {
 	window.scanInfo.Media = scannedMedia
 }
 
+// Panic handles the gui crashes
+func (window *Window) Panic(err error) {
+	log.Debug("Panic")
+	displayErrorDialog(err)
```
Comment (mdhorn) [inline gui/window.go]: Should this move after the log.RequestCrashInfo()? To ensure we log the telemetry and the crash info before the user has a chance to dismiss and close the app?

File: gui/window.go [hunk 2]
```diff
@@ -800,3 +811,39 @@ func GetThankYouMessage() string {
...
+	if err != nil {
+		log.Error("Error creating label", err)
+		return
+	}
+	label.SetHAlign(gtk.ALIGN_CENTER)
+	label.SetMarginBottom(common.TopBottomMargin)
+	label.SetSelectable(true)
+	contentBox.PackStart(label, true, true, 0)
+
+	title := utils.Locale.Get("Sorry, something went wrong...")
```
Comment (mdhorn) [inline gui/window.go]: Maybe we just say "Something went wrong"? Debating the "Sorry" portion.

File: log/log.go [hunk 3]
```diff
@@ -86,26 +87,23 @@ func SetOutputFilename(logFile string) (*os.File, error) {
 	return filehandle, nil
 }
 
+// GetCrashInfoMsg returns the crash info message.
+func GetCrashInfoMsg() string {
+	msg := utils.Locale.Get("Please report this crash using %s", "GitHub Issues:")
+	msg += "\n" + "https://github.com/clearlinux/clr-installer/issues"
+	msg += "\n\n" + "Include the following as attachments to enable diagnosis:"
```
Comment (mdhorn) [inline log/log.go]: Should we not localize all of these sentences?

File: log/log.go [hunk 4]
```diff
@@ -86,26 +87,23 @@ func SetOutputFilename(logFile string) (*os.File, error) {
...
+	msg += "\n" + preConfName
+	msg += "\n" + logFileName
+	msg += "\n\n" + "You may need to remove any personal data of concern from the attachments."
```
Comment (mdhorn) [inline log/log.go]: Should we not localize all of these sentences?

File: gui/window.go (hunk 1 above)
Comment (reaganlo) [reply gui/window.go]: Done.

File: gui/window.go (hunk 2 above)
Comment (reaganlo) [reply gui/window.go]: Done.

File: log/log.go (hunk 3 above)
Comment (reaganlo) [reply log/log.go]: Yeah we should. Completely missed out the translations during rebase.
Good catch.

File: log/log.go (hunk 4 above)
Comment (reaganlo) [reply log/log.go]: Done.

**Comment by reaganlo** [comment]
@mdhorn I just modified the code to manually return an errors.Errorf("TEST") when err == nil where Panic is called.
Not a perfect test.

**Comment by reaganlo** [comment]
@mdhorn Made the changes for error popup font color and pre-check page translation fix.

**Pull request merged**